packages/shared/src/index.ts

```diff
@@ -0,0 +1 @@
+export * from "./internal.js";
```

Comment:
Bleh, I submitted my review and my comment here was not included, letting you know just in case the comment suddenly pops here again.

Is this expected? To expose all `internal.js` members outside the module?

Reply:
in this case I did it like this because all of the shared things should be exposed. A bit of lazyness to manually export every single thing from shared again in 'external.ts' 😅